src/components/app/overrideGlobalLocalStorage.tsx

```diff
@@ -38,13 +38,18 @@ export function OverrideGlobalLocalStorage() {
           removeItem: () => null,
           setItem: () => null,
         }
-        window.localStorage = localStorageNoop
-        Object.defineProperty(window, 'localStorage', {
-          value: localStorageNoop,
-          configurable: true,
-          enumerable: true,
-          writable: false,
-        })
+        // Try to assign directly first
+        try {
+          window.localStorage = localStorageNoop
+        } catch (directAssignError) {
+          // If direct assignment fails, use defineProperty but with writable: true
+          Object.defineProperty(window, 'localStorage', {
+            value: localStorageNoop,
+            configurable: true,
+            enumerable: true,
+            writable: true,
+          })
+        }
```

Comment:
Wish I didn't have a np for the first PR fully made with AI

Let's create an aux function to do this assign, exact same code block from line 41
```suggestion
        setWindowLocalStorage(localStorageNoop)
        // ...
        setWindowLocalStorage(sessionStorage)
```